src/pages/organize/[orgId]/projects/[campId]/areaassignments/[areaAssId]/report.tsx

```diff
@@ -94,216 +111,352 @@ const AreaAssignmentReportPage: PageWithLayout<AreaAssignmentReportProps> = ({
-          )}
-        </ZUIFuture>
-      </Box>
-    </>
+            </Box>
+          </Box>
+          <Box ml={2} width="40%">
+            <ZUICard
+              header="Successful visit"
+              subheader=" Question to use to count a visit as successful."
+              sx={{ mb: 2 }}
+            >
+              <Divider />
+              <FormControl fullWidth sx={{ mt: 2 }}>
+                <InputLabel>Defines success</InputLabel>
+                <Select
+                  disabled={
+                    assignment.metrics.filter(
+                      (metric) => metric.kind === 'boolean'
+                    ).length <= 1
+                  }
+                  label="Defines success"
+                  onChange={(ev: SelectChangeEvent) => {
+                    const updatedMetrics =
+                      areaAssignmentFuture.data?.metrics.map(
+                        (m: ZetkinMetric) => {
+                          if (m.id === ev.target.value) {
+                            return {
+                              ...m,
+                              definesDone: true,
+                            };
+                          } else {
+                            return {
+                              ...m,
+                              definesDone: false,
+                            };
+                          }
+                        }
+                      );
+
+                    updateAreaAssignment({
+                      metrics: updatedMetrics,
+                    });
+                  }}
+                  value={
+                    assignment.metrics.find((metric) => metric.definesDone)
+                      ?.id || ''
+                  }
+                >
+                  {assignment.metrics.map((metric) =>
+                    metric.kind === 'boolean' ? (
+                      <MenuItem key={metric.id} value={metric.id}>
+                        {metric.question}
+                      </MenuItem>
+                    ) : null
+                  )}
+                </Select>
+              </FormControl>
+            </ZUICard>
+            <ZUICard
+              header="Data precision & privacy"
+              subheader="Configuring where to store the data is a matter of striking a
+                balance between precision and privacy that is right for your
+                cause"
+              sx={{ mb: 2 }}
+            >
+              <Divider />
+              <FormControl>
+                <RadioGroup
+                  onChange={(ev) => {
+                    const value = ev.target.value;
+                    if (value === 'household' || value === 'location') {
+                      updateAreaAssignment({
+                        reporting_level: value,
+                      });
+                    }
+                  }}
+                  value={assignment.reporting_level}
+                >
+                  <Typography mt={1}>Collect data..</Typography>
```

Comment:
Needs to be internationalized, but also needs an extra dot (three dots) at the end.